docker/install.sh

```diff
@@ -19,11 +19,10 @@
 
 set -ex
 
-source /opt/ros/kinetic/setup.bash
 source /opt/autoware.ai/ros/install/setup.bash --extend
 
 cd ~/carma_ws
-./src/carma-platform/carma_build.bash -c ~/carma_ws -a /opt/autoware.ai/ -x -m "-DCMAKE_BUILD_TYPE=Release"
+./src/carma-platform/carma_build.bash -c ~/carma_ws -a /opt/autoware.ai/ -x -m "-DCMAKE_BUILD_TYPE=Debug"
```

Comment:
This should be kept as Release

Reply:
Yes. We currently have a crash that we need to debug w/ the Release version. Will debug that crash and fix this before merging.

Reply:
Crash fixed and script changed back to Release.